tests/sanity.cpp

```diff
@@ -180,16 +180,7 @@ START_TEST(basic)
 
     print_dbg("buffer size: %zu\n", size);
     CUdeviceptr d_A;
-    ASSERTDRV(cuMemAlloc(&d_A, size));
-
-    unsigned int flag = 1;
-    ASSERTDRV(cuPointerSetAttribute(&flag, CU_POINTER_ATTRIBUTE_SYNC_MEMOPS, d_A));
-
-    uint32_t *init_buf = new uint32_t[size];
-    uint32_t *copy_buf = new uint32_t[size];
-
-    init_hbuf_walking_bit(init_buf, size);
-    memset(copy_buf, 0, sizeof(*copy_buf) * sizeof(uint32_t));
```

Comment:
this buffer initialization code has been removed, was it useless?

Reply:
This "basic" unit test is for testing gdr_pin_buffer -- no mapping, no data copied. You can see that copy_buf and init_buf were not used anywhere.

Reply:
thanks for cleaning it up